specification/paths/CombinedFiles.json

```diff
@@ -76,7 +77,7 @@
   },
   "post": {
     "tags": [
-      "CombinedFiles"
+      "RPC"
```

Comment:
I think we need to keep this (while our app still uses it) and we need to introduce the RPC ass a new endpoint.
Since the RPC format is always `<verb>-<noun>` it can be `POST /combine-files` so it's not a duplicate route.